lib/acts_as_taggable_on_mongoid/taggable/tag_type_definition/names.rb

```diff
@@ -10,6 +10,18 @@ def tag_list_name
           @tag_list_name ||= "#{single_tag_type}_list"
         end
 
+        def from_list_name
+          @tag_list_name ||= "#{tag_type.to_s.pluralize}_from"
```

Comment:
Naming/MemoizedInstanceVariableName: Memoized variable `@tag_list_name` does not match method name `from_list_name`. Use `@from_list_name` instead.